functions.php

```diff
@@ -14,7 +16,11 @@ function db_connect() {
   }
 
 
-
+  /**
+  * Вовращает остаток времени до будущей даты и добавляет к строке 0, если остаток часов или минут меньше 10
+  *
+  * @param $date будущая дата в формате '2020-10-15'
```

Comment:
```suggestion
  * @param string $date будущая дата в формате '2020-10-15'
```
Стоит добавлять типы